src/components/protocol_handler/src/incoming_data_handler.cc

```diff
@@ -50,18 +50,18 @@ static const size_t MIN_HEADER_SIZE =
 
 ProtocolFramePtrList IncomingDataHandler::ProcessData(
     const RawMessage& tm_message,
-    RESULT_CODE* result,
+    RESULT_CODE& result,
     size_t* malformed_occurrence) {
   LOG4CXX_AUTO_TRACE(logger_);
-  DCHECK(result);
+  // DCHECK(result);
```

Comment:
@SKobziev i think this commented code should be removed

Reply:
This comment is removed.